sample_data_from_verso/data/profiles/BIBFRAME 2.0 Agents Contribution.json

```diff
@@ -87,10 +87,10 @@
                 ]
             }
         ],
-        "id": "profile:bf2:AgentsContribution",
+        "id": "bf2:AgentsContribution",
```

Comment:
Is this identifier change from them?  In the BFF, I straight-out made up the "convention" of "resourceTemplate:xxx:xxx"  instead of "profile:xxx:xxx".   We probably need to get a "ruling" on what these identifiers will be.

Reply:
Yes, this identifier change is from Verso; it seems they only changed some of the ids.